src/python_be.h

```diff
@@ -281,6 +281,9 @@ class ModelInstanceState : public BackendModelInstance {
   std::unique_ptr<boost::asio::thread_pool> thread_pool_;
   std::unordered_map<void*, std::shared_ptr<InferPayload>> infer_payload_;
   std::unique_ptr<RequestExecutor> request_executor_;
+  std::mutex response_factory_map_mutex_;
+  std::unordered_map<intptr_t, TRITONBACKEND_ResponseFactory*>
+      response_factory_map_;
```

Comment:
Response factory doesn't seem to be used?

Reply:
It is used for decoupled models. e.g. https://github.com/triton-inference-server/python_backend/pull/319/files#diff-ca67ec8693bbdfbd82a858e307f87fa39fc9187d39228f1afc5375837dc3f60dR378